packages/jest/src/automatic.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { AxeResults } from '@sa11y/common';
+import { getViolationsJSDOM } from '@sa11y/assert';
+import { A11yError } from '@sa11y/format';
+
+/**
+ * Options for Automatic checks to be passed to {@link registerSa11yAutomaticChecks}
+ */
+export type AutoCheckOpts = {
+    runAfterEach?: boolean;
+    cleanupAfterEach?: boolean;
+    // TODO (feat): add support for result consolidation across all tests for a test run
```

Comment:
Should we capture these TODOs as issues?

Reply:
Thanks Trevor, Yes that is a good idea. 
* I have been thinking of exploring [GitHub Discussions](https://docs.github.com/en/discussions) to capture TODOs so that Issues are focussed on bugs etc. What do you think about that?
* Looks like there are also tools to convert TODOs in code to issues automatically e.g. [TODO to Issue · Actions · GitHub Marketplace · GitHub](https://github.com/marketplace/actions/todo-to-issue). I will explore them more.

Reply:
Cloned discussion to https://github.com/salesforce/sa11y/issues/55